src/path.js

```diff
@@ -23,12 +23,29 @@ function appendRound(digits) {
   };
 }
 
+function equal(x0, y0, x1, y1) {
+  return x0 === x1 && y0 === y1;
+}
+
+function equalRound(digits) {
+  const d = Math.floor(digits);
+  if (d > 15) return equal;
+  const k = 10 ** d;
+  const r = function(value) {
+    return Math.round(value * k);
+  };
+  return function(x0, y0, x1, y1) {
+    return r(x0) === r(x1) && r(y0) === r(y1);
+  };
+}
+
 export class Path {
   constructor(digits) {
     this._x0 = this._y0 = // start of current subpath
     this._x1 = this._y1 = null; // end of current subpath
     this._ = "";
     this._append = digits == null ? append : appendRound(digits);
+    this._equal = digits === null ? equal : equalRound(digits);
```

Comment:
```suggestion
    this._equal = digits == null ? equal : equalRound(digits);
```

Use loose equals to match previous line.